crypto/src/main/java/org/web3j/crypto/BlobUtils.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2024 Web3 Labs Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.web3j.crypto;
+
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+
+import ethereum.ckzg4844.CKZG4844JNI;
+
+import org.web3j.abi.datatypes.Bytes;
+
+public class BlobUtils {
```

Comment:
There is an inconsistency in this util class related to static and non-static methods.
I would keep everything here as static method and send Blob every time as an argument when you run these helper methods. Else you might end having code as bellow, and it doesn't look good:
BlobUtils.method();
BlobUtils util = new BlobUtils(blob)

Reply:
Makes sense, thanks. Will implement all static methods